src/parser/monk/windwalker/modules/spells/azeritetraits/FuryOfXuen.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import Events from 'parser/core/Events';
+import Analyzer, { SELECTED_PLAYER_PET, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import AbilityTracker from 'parser/shared/modules/AbilityTracker';
+import {calculateAzeriteEffects} from 'common/stats';
+import SPELLS from 'common/SPELLS';
+import TraitStatisticBox, { STATISTIC_ORDER } from 'interface/others/TraitStatisticBox';
+import { formatNumber, formatPercentage } from 'common/format';
+
+const furyOfXuenStats = traits => Object.values(traits).reduce((obj, rank) => {
+  const [haste] = calculateAzeriteEffects(SPELLS.FURY_OF_XUEN.id, rank);
+  obj.haste += haste;
+  return obj;
+}, {
+  haste: 0,
+});
+
+export const STAT_TRACKER = {
+  haste: combatant => furyOfXuenStats(combatant.traitsBySpellId[SPELLS.FURY_OF_XUEN.id]).haste,
+};
+
+/**
+ * Your Combo Strikes grant a stacking 2% chance for your next Fists of Fury to grant 768 Haste and invoke Xuen, The White Tiger for 8 sec.
+ * 
+ * The tooltip is rounded up and it actually gives 1.5% per stack
+ */
+class FuryOfXuen extends Analyzer {
+  static dependencies = {
+    abilityTracker: AbilityTracker,
+  };
+
+  damageDone = 0;
+  furyXuens = [];
+  stacksGained = 0;
```

Comment:
These two variables are unused. If you want to implement it later, that's fine, but for now - either do it in this PR or remove it.